styles.css

```diff
@@ -218,7 +218,7 @@ h4 {
   flex-grow: 0;
 }
 
-.card-work-row1 {
+.grid-work-row1 {
```

Comment:
- [ ] We actually want you to use `display:grid` for the main work section. Kindly ensure to improve upon this aspect.

Reply:
Changed to display: grid